bin/runner.js

```diff
@@ -6,7 +6,11 @@ const url = require('url');
 
 const args = minimist(process.argv.slice(2));
 if (args._.length !== 2 || args.h || args.help) {
-  process.stderr.write('usage: sparql-benchmark-runner endpoint-url path-to-queries [--replication 5] [--warmup 0] [--output output.csv]\n');
+  process.stderr.write('usage: sparql-benchmark-runner endpoint-url path-to-queries [-r 5] [-w 0] [-o output.csv] [--timestamps]\n');
+  process.stderr.write('  -r # of replication runs\n');
+  process.stderr.write('  -w # of warmup runs\n');
+  process.stderr.write('  -o output file\n');
+  process.stderr.write('  --timestamps if enabled also outputs timestamps for when each result was received\n');
```

Comment:
Perhaps you can use `-t` here for consistency?

Reply:
yea was doubting, wanted to make it special since it's a boolean value instead of one with an additional parameter

Reply:
I have no strong opinion, either way is fine for me :-)